sentry.properties

```diff
@@ -0,0 +1,4 @@
+defaults.project=openfoodfacts-android
+defaults.org=openfoodfacts
+auth.token=4664a0f9a08e4b2facef49f778b82321993e7e8b0783439fa81f7741b36249b3
```

Comment:
@philippeauriach Does this token need to be public in Git(Hub)? According to https://github.com/getsentry/sentry-react-native/issues/112#issuecomment-309980345, it's only used to upload symbols to Sentry. Therefore, it's only required for releases. With the token in Git, everyone would be able to publish new symbols (potentially overriding existing version?) to Sentry.

Reply:
It is not mandatory to make it public of course, it is only used during assemble to upload symbols as you mentioned. It is possible to use an environment variable : `export SENTRY_AUTH_TOKEN=your-auth-token` . Making the change

Reply:
I was just wondering if this might be a tiny security issue. 🙂

Reply:
You're right, I updated the code, and invalidated the commited token. A new one has been generated and is set on @teolemon computer. I updated the ios sentry branch too